### ISSUE TITLE
Access current value from the Story

### ISSUE THREAD

**Issue by colorfulfool** [le0pard/storybook-addon-root-attribute#23]
Hi,

Is there a way to get the currently selected value of root attribute from within a story?

Something like:
```tsx
...
  rootAttribute: {
    defaultState: {
      name: "Dark",
      value: "dark"
    },
    states: [
      {
        name: "Light",
        value: null
      }
    ]
  },
...
const Template: Story = (state, context) => (
  context.rootAttribute // returns "dark"
);
```

**Comment by le0pard** [comment]
@colorfulfool how about read it from DOM?

**Comment by colorfulfool** [comment]
Should be possible with a mutation observer, I guess. But it the app itself the theme (dark/light) is placed in Redux, from where it's synced to DOM. Appearance is mostly influenced by CSS that the DOM attribute triggers, but some components access the theme directly. So it would make sense to be able to inject the current value into Redux?

**Comment by le0pard** [comment]
> So it would make sense to be able to inject the current value into Redux?

Yes, it make sense. Also some devs using context to pass "theme" to components

**Comment by colorfulfool** [comment]
So, is it possible?
